code/code/disc/disc_brawling.cc

```diff
@@ -29,9 +33,13 @@ CDBrawling::CDBrawling(const CDBrawling &a) :
   skStomp(a.skStomp),
   skBrawlAvoidance(a.skBrawlAvoidance),
   skBodyslam(a.skBodyslam),
+  skHeadbutt(a.skHeadbutt),
+  skKneestrike(a.skKneestrike),
   skSpin(a.skSpin),
   skCloseQuartersFighting(a.skCloseQuartersFighting),
-  skTaunt(a.skTaunt)
+  skTaunt(a.skTaunt),
+  skTrip(a.skTrip),
+  skAdvBerserk(a.skAdvBerserk)
 {
 }
 
```

Comment:
These copy constructors and assignment operators usually don't do anything new that the compiler wouldn't generate automatically. We should get rid of them all, tho maybe not in this  PR.

Reply:
I've tested removing all those constructors from about a dozen disc_x.cc files and letting them be replaced by implicit constructors and the code compiled fine, so I think you're right that they're not needed. One big PR getting rid of them all is on my to-do list.

Reply:
Happy to help if I can with removing them/testing, but would prefer to do so in a new PR.